_component_design/tables-basic.md

```diff
@@ -10,9 +10,158 @@ resource: true
+
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- Keep the column count to under seven. 
+- Keep visual clutter to a minimum to maintain a users focus.
+- When organizing content in a table, focus on the user’s primary task.
+- Preserve hierarchies from left to right. (Most important to least important)
+- Providing filtering and sorting table controls allows users to interact with an select the most important information.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the table pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Default layout
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Include the check-box icon if a user needs to select or manipulate data within a table. This will always be the first element in a table row, see [table controls]({{site.baseurl}}/components/tables-controls.html)
+- Include the action-cog icon if a user requires the trigger two or more actions on a row.
+- Text justification within each cell is content specific.
+  - Text should be left justified.
+  - Icons, “&mdash;” (null/undefined), and “n/a” (not applicable) should be center justified.
+  - Numbers should be right justified.
+- Only use units in row if they differ row by row and cannot be standardized. Units in body rows appear without parentheses.
+- Truncated text should appear in a [tooltip]({{site.baseurl}}/components/tooltip.html), enabling display on hover or click.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-default.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Condensed table
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The use of a condensed table provides for views that display either many or few results. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Condensed table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-condensed.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Two-row / small tables
+{: .hxSubSectionTitle}
+
+For instance, when a small amount of information, that is best presented in a tabular format, is presented, the condensed table format allows for a concise presentation. 
```

Comment:
This sentence has some repetition and seems a little disconnected.  Just a wag:

Use the condensed table style for small sets of data which are best represented in tabular form.
In cases like this, tables are preferred over bullet lists, or key value lists.

Reply:
I like it.